pkg/operator/ceph/csi/ceph_connection.go

```diff
@@ -28,23 +28,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func CreateUpdateCephConnection(c client.Client, clusterInfo *cephclient.ClusterInfo, clusterSpec cephv1.ClusterSpec) error {
+func CreateUpdateCephConnection(c client.Client, clusterInfo *cephclient.ClusterInfo, clusterSpec cephv1.ClusterSpec, operatorNamespace string) error {
 
 	logger.Infof("Configuring ceph connection CR %q in namespace %q", clusterInfo.NamespacedName().Name, clusterInfo.NamespacedName().Namespace)
 	csiCephConnection := &csiopv1a1.CephConnection{}
 
 	csiCephConnection.Name = clusterInfo.NamespacedName().Name
-	csiCephConnection.Namespace = clusterInfo.NamespacedName().Namespace
+	csiCephConnection.Namespace = operatorNamespace
```

Comment:
Instead of passing around the operator namespace everywhere, we conveniently have an env var that can be called directly here:
```
os.Getenv(k8sutil.PodNamespaceEnvVar)
```